chapters/statemachines.tex

```diff
@@ -31,7 +31,7 @@ \chapter{State Machines}\label{state-machines}
   indirectly with appropriate conditions on transitions.
 \item
   Modelica blocks can be used as states. They might contain
-  clocked or clocked discretized continuous-time equations (in the
+  clocked or discretized equations (in the
```

Comment:
Isn't a discretized equation a clocked equation?

Reply:
Yes. But it will require more rewriting, I will commit that.

Reply:
I consider it resolved with the change, but note that it is a new change.